src/pdf_beamtime/src/pdf_beamtime_server.cpp

```diff
@@ -556,7 +582,14 @@ void PdfBeamtimeServer::execute_stop()
 
 void PdfBeamtimeServer::handle_abort()
```

Comment:
Abort should attempt necessary cleanup.

Reply:
Stop and abort now handles necessary cleanup.
https://github.com/maffettone/erobs/blob/d76de5584bf62ca3b3e65c1eac84eb61810aa109/src/pdf_beamtime/src/pdf_beamtime_server.cpp#L578-L583

https://github.com/maffettone/erobs/blob/d76de5584bf62ca3b3e65c1eac84eb61810aa109/src/pdf_beamtime/src/pdf_beamtime_server.cpp#L548-L553